src/utils/deleteFileFromPath/deleteFileFromPath.ts

```diff
@@ -13,15 +13,15 @@ const DELETION_INCREMENT = 1;
 
 const recursivelySetDeletedOrder = (
   treeCore: Core[],
-  newDeletedOrder: number,
+  newDeletedOrder: number
 ): void => {
   for (let index = 0; index < treeCore.length; index += 1) {
     if (treeCore[index].deletedOrder === -1) {
```

Comment:
-1 should be a constant. Try to fix other peoples code as you go for this project for learning purposes.